spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/tf/Fill.scala

```diff
@@ -15,38 +15,96 @@
  */
 package com.intel.analytics.bigdl.nn.tf
 
-import com.intel.analytics.bigdl.nn.abstractnn.TensorModule
-import com.intel.analytics.bigdl.tensor.Tensor
+import com.google.protobuf.ByteString
+import com.intel.analytics.bigdl.nn.Utils
+import com.intel.analytics.bigdl.nn.abstractnn.{AbstractModule, TensorModule}
+import com.intel.analytics.bigdl.tensor._
 import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.Table
 
 import scala.reflect.ClassTag
 
 /**
  * Creates a tensor filled with a scalar value. Input should be a 1-D tensor defining
  * the shape of the output tensor.
- * @param value the scalar value to be filled.
  */
 @SerialVersionUID(-471757174144422555L)
-private[bigdl] class Fill[T: ClassTag](value: T) (implicit ev: TensorNumeric[T])
-  extends TensorModule[T] {
+private[bigdl] class Fill[T: ClassTag]() (implicit ev: TensorNumeric[T])
```

Comment:
value B?

Reply:
B is inferenced from input, we do not know B in advance